build/Jenkinsfile

```diff
@@ -1,80 +1,107 @@
 #!/usr/bin/env groovy
 
-podTemplate(label: 'jenkins-slave', 
+podTemplate(label: 'jenkins-slave',
     containers: [
         containerTemplate(
             name: 'mattermost-mysql',
             image: 'mysql:5.7',
             alwaysPullImage: false,
-            resourceRequestCpu: '1000m',
-            resourceLimitCpu: '2000m',
-            resourceRequestMemory: '2Gi',
-            resourceLimitMemory: '4Gi',
+            resourceRequestCpu: '500m',
+            resourceLimitCpu: '1000m',
+            resourceRequestMemory: '1Gi',
+            resourceLimitMemory: '2Gi',
             ports: [portMapping(name: 'mysql', containerPort: 3306, hostPort: 3306)],
-            envVars: [envVar(key: 'MYSQL_ROOT_PASSWORD', value: 'mostest'), envVar(key: 'MYSQL_USER', value: 'mmuser'), envVar(key: 'MYSQL_PASSWORD', value: 'mostest'), envVar(key: 'MYSQL_DATABASE', value: 'mattermost_test')]
+            envVars: [
+                envVar(key: 'MYSQL_ROOT_PASSWORD', value: 'mostest'),
+                envVar(key: 'MYSQL_USER', value: 'mmuser'),
+                envVar(key: 'MYSQL_PASSWORD', value: 'mostest'),
+                envVar(key: 'MYSQL_DATABASE', value: 'mattermost_test')
+            ]
         ),
         containerTemplate(
-            name: 'golang', 
-            image: 'golang:1.8', 
-            ttyEnabled: true, 
+            name: 'golang',
+            image: 'golang:1.8',
+            ttyEnabled: true,
             command: 'cat',
-            alwaysPullImage: false,
+            alwaysPullImage: true,
```

Comment:
Wouldn't it be better to specify the exact version with the tag (e.g. "golang:1.8.0") and _not_ always pull the image? I think we want any Go upgrades to be premeditated, and if we specify the exact version, there's no point in always pulling. Also, we're on Go 1.9.1 now. So I think this should probably be...

```
name: 'golang',
image: 'golang:1.9.1'
alwaysPullImage: false,
```

Reply:
I forgot to revert that, my bad. 
done